app/models/transfer_request.rb

```diff
@@ -7,6 +7,10 @@ class TransferRequest < ApplicationRecord
   include AASM
   extend Request::Statemachine::ClassMethods
 
+  # Determines if we attempt to filter out {Aliquot#equivalent? equivalent} aliquots
+  # before performing transfers.
+  attr_accessor :merge_equivalent_aliquots
```

Comment:
Where does `:merge_equivalent_aliquots` get set? By Limber's API call?

Reply:
By Limber API, so we had to define it in sequencescape-client-api:
https://github.com/sanger/sequencescape-client-api/commit/e934b023d186394e751b0834a93502f489ce3b04
And in Limber as well, when the pull request for limber is ready